combat/Combat.tscn

```diff
@@ -177,7 +177,22 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="Ticks" type="Label" parent="CanvasLayer/Background"]
+[node name="AnimatedSprite" type="AnimatedSprite" parent="CanvasLayer/Background"]
+position = Vector2( 320, 91 )
+frames = SubResource( 2 )
+animation = "satelite"
+frame = 88
+playing = true
+
+[node name="DebugContainer" type="Container" parent="CanvasLayer"]
+visible = false
```

Comment:
Why is this false?

Reply:
Defaulting the debug container to not be visible. We turn it on if you have debug mode `TRACE`.

Reply:
Oh. Miss saw. Thought it was visible false on the animation sprite.